src/index.js

```diff
@@ -71,7 +72,8 @@ const Reliable = or(
   WebRTCStar,
   WebRTCDirect,
   TCP,
-  UTP
+  UTP,
+  DNSADDR
```

Comment:
Was this added to Reliable to support the test addition? We may need to replace this with DNS to cover the rest of the formats.

Reply:
This was added to support dials using `dnsaddr/p2p/{ID}` which fails the mafmt test for `P2P` on dial. I will change to support all DNS to cover the remaining cases